compose/src/main/java/com/bottlerocketstudios/compose/snippets/SnippetUiModel.kt

```diff
@@ -1,7 +1,9 @@
 package com.bottlerocketstudios.compose.snippets
 
+import com.bottlerocketstudios.compose.util.StringIdHelper
+
 data class SnippetUiModel(
     val title: String,
     val userName: String,
-    val updatedTime: String
+    val formattedTime: StringIdHelper,
```

Comment:
@VanyaHuamanBottleRocket To help clarify what time this represents, how about refactoring the property to be something like `formattedLastUpdatedTime`